lib/libvarnish/vsa.c

```diff
@@ -173,6 +173,24 @@ struct suckaddr {
 
 const int vsa_suckaddr_len = sizeof(struct suckaddr);
 
+static const struct suckaddr ipv4_any = { .magic = 0x4b1e9335,
+					 .sa4 = { .sin_family = AF_INET,
+						  .sin_addr.s_addr = INADDR_ANY,
+						  .sin_port = 0 }
+};
+static const struct suckaddr ipv6_any = { .magic = 0x4b1e9335,
+					 .sa6 = { .sin6_family = AF_INET6,
+						  .sin6_addr = IN6ADDR_ANY_INIT,
+						  .sin6_port = 0 }
+};
+
+const struct suckaddr *vsa_ipv4_any() {
+	return (& ipv4_any);
+}
+const struct suckaddr *vsa_ipv6_any() {
+	return (& ipv6_any);
+}
```

Comment:
Instead of functions, `ipv?_any` symbols could be exposed similarly to how `vsa_suckaddr_len` is (and prefixed with `vsa_`).